modules/users/client/views/profile/profile-view-basics.client.view.html

```diff
@@ -15,6 +15,23 @@ <h4 class="profile-username">@{{ profileCtrl.profile.displayUsername || profileC
       <span ng-if="profileCtrl.profile.gender" ng-class="{ 'text-capitalize': !profileCtrl.profile.birthdate }">{{ profileCtrl.profile.gender }}.</span>
     </div>
 
+    <!-- Reply Rate -->
+    <div class="profile-sidebar-section">
+      <i class="icon-fw icon-reply text-muted"></i>
+      <span ng-if="profileCtrl.profile.replyRate">
+        Replied to
+        <span ng-bind="profileCtrl.profile.replyRate"></span>
+      </span>
```

Comment:
Use [one-time binding](https://docs.angularjs.org/guide/expression#one-time-binding) here so we don't get extra watchers for something that's not gonna change.